tls/ttls.c

```diff
@@ -1262,10 +1262,16 @@ ttls_handshake_free(TlsHandshake *hs)
 
 	crypto_free_shash(hs->desc.tfm);
 
+	if (!IS_ERR_OR_NULL(ci)) {
+		if (ttls_ciphersuite_uses_ecdh(ci) ||
+		    ttls_ciphersuite_uses_ecdhe(ci))
+			ttls_ecdh_free(&hs->ecdh_ctx);
```

Comment:
Usually we use curve brackets if the `if` statement takes more than one line.

Reply:
Added.